azure-boards-common/src/main/java/com/synopsys/integration/azure/boards/common/service/state/AzureWorkItemTypeStateService.java

```diff
@@ -40,6 +40,13 @@
     public static final AzureSpecTemplate API_SPEC_ORGANIZATION_PROCESS_WORKITEMTYPE_STATES = new AzureSpecTemplate("/{organization}/_apis/work/processes/{processId}/workItemTypes/{witRefName}/states");
     public static final AzureSpecTemplate API_SPEC_ORGANIZATION_PROCESS_WORKITEMTYPE_STATE_INDIVIDUAL = new AzureSpecTemplate("/{organization}/_apis/work/processes/{processId}/workItemTypes/{witRefName}/states/{stateId}");
 
+    public static final String PATH_ORGANIZATION_REPLACEMENT = "{organization}";
+    public static final String PATH_PROJECT_REPLACEMENT = "{project}";
+    public static final String PATH_TYPE_REPLACEMENT = "{type}";
+    public static final String PATH_PROCESS_ID_REPLACEMENT = "{processId}";
+    public static final String PATH_WIT_REF_NAME_REPLACEMENT = "{witRefName}";
+    public static final String PATH_STATE_ID_REPLACEMENT = "{stateId}";
```

Comment:
Unfortunately, the whole point of the template string is to be able to nicely read the template. Otherwise there would be no real purpose for the class.